ingestion/src/metadata/utils/connections.py

```diff
@@ -323,3 +327,28 @@ def _(connection: DeltaLakeClient) -> None:
         raise SourceConnectionException(
             f"Unknown error connecting with {connection} - {err}."
         )
+
+
+@get_connection.register
+def _(connection: RedashConnection, verbose: bool = False):
+
+    from redash_toolbelt import Redash
+
+    try:
+        redash = Redash(connection.hostPort, connection.apiKey)
+        redash_client = RedashClient(redash)
+        return redash_client
+
+    except Exception as err:
+        logger.error(f"Failed to connect with error :  {err}")
+        logger.error(err)
+
+
+@test_connection.register
+def _(connection: RedashClient) -> None:
```

Comment:
As redash already ships the `Redash` class I don't think we need to wrap it inside the `RedashClient`. That should simplify the code a bit

Reply:
Importing Redash directly in `connections.py` might break ingestion in case if we have not installed the redash package and we are trying to test other connectors for example `mysql`.
@pmbrull

Reply:
oh you're right! I think I already asked this multiple times 😅  sorry about that. If it is not there, could you please add some docstring in `connection_clients.py` explaining this? Thanks 🙏